src/System.Diagnostics.Tracing/ref/System.Diagnostics.Tracing.cs

```diff
@@ -54,9 +54,10 @@ public partial class EventCommandEventArgs : System.EventArgs
         public bool DisableEvent(int eventId) { throw null; }
         public bool EnableEvent(int eventId) { throw null; }
     }
-    public class EventCounter {
+    public class EventCounter : IDisposable {
         public EventCounter(string name, EventSource eventSource) { }
         public void WriteMetric(float value) { }
+        public void Dispose() { }
```

Comment:
This is public surface area, right?  We don't need to review this in an API review?